### PR TITLE
MealsharingNET_project-final

### PR DESCRIPTION
**Pull request by seenachacko**

### CODE REVIEW

File: final/MealsharingNET/Controllers/MealController.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+using Microsoft.AspNetCore.Mvc;
+using MealsharingNET.Models;
+namespace MealsharingNET.Controllers;
+
+[ApiController]
+[Route("meals")]
+public class MealController : ControllerBase{
```
Comment (calindurnea) [inline final/MealsharingNET/Controllers/MealController.cs]: Try to use the auto formatting functionality to make the code more readable :) It's Alt + Shift+ F I think

File: final/MealsharingNET/Controllers/UserController.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+[ApiController]
+[Route("users")]
+
+public class UserController : ControllerBase{
+  private IUserRepository _repo;
+  public UserController(IUserRepository repo){
+      _repo=repo;
+  }
+    [HttpGet("ListAllUsers")]
+    public async Task <List<User>> ListAllUser(){
```
Comment (calindurnea) [inline final/MealsharingNET/Controllers/UserController.cs]: There is no need to have a space after Task here, `Task<List<User>>` is the return type altogether

File: final/MealsharingNET/Controllers/WeatherForecastController.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
+using Microsoft.AspNetCore.Mvc;
```
Comment (calindurnea) [inline final/MealsharingNET/Controllers/WeatherForecastController.cs]: You can remove this file if it is not used

File: final/MealsharingNET/MealRepository.cs [hunk 4]
```diff
@@ -0,0 +1,36 @@
+namespace MealsharingNET.Services;
+using System.Collections.Generic;
+using MealsharingNET.Models;
+using MySql.Data.MySqlClient;
+using Dapper;
```
Comment (calindurnea) [inline final/MealsharingNET/MealRepository.cs]: The usings are usually above the namespace

File: final/MealsharingNET/Models/User.cs [hunk 5]
```diff
@@ -0,0 +1,9 @@
+namespace MealsharingNET;
```
Comment (calindurnea) [inline final/MealsharingNET/Models/User.cs]: The namespace here should contain `Models`, as the class is in the Models folder

File: final/MealsharingNET/WeatherForecast.cs [hunk 6]
```diff
@@ -0,0 +1,12 @@
+namespace MealsharingNET;
```
Comment (calindurnea) [inline final/MealsharingNET/WeatherForecast.cs]: You can remove this file if it is not used

**Review by calindurnea — COMMENTED** [review]
Very nice!
I left a few comments, let me know if you have any questions or if i should explain anything 😊

**Comment by seenachacko** [comment]
> Very nice! I left a few comments, let me know if you have any questions or if i should explain anything 😊

Thanks for the feedback

**Pull request merged**